src/pod-security-policy/capabilities/src_test.rego

```diff
@@ -103,6 +103,21 @@ test_input_drop_undefined_x2 {
    results := violation with input as input
    count(results) == 2
 }
+test_input_drop_literal_all {
+   input := { "review": input_review([cdrop(["all"])]), "parameters": {"requiredDropCapabilities": ["one", "two"]}}
```

Comment:
`ALL` instead of `all` here and below?

Reply:
I have now made the "ALL" comparison case insensitive.

Reply:
Just to verify... kubernetes interprets "ALL" in a case-insensitive manner?